doc/linux.t2t

```diff
@@ -59,6 +59,16 @@ See http://qgis.org/en/site/forusers/alldownloads.html#debian-ubuntu for
 currently supported distributions (plain xenial's GDAL for instance is to old
 and we build with GDAL2 from ubuntugis).
 
+If you plan to code, you will need these packages:
+```
+apt install flip python-autopep8
+```
+
+And also a perl library:
+```
+sudo cpan YAML::Tiny
```

Comment:
Isn't it better to install this from the repos?

Reply:
I tried `sudo apt install libconfig-yaml-perl` but it was not working. Only this cpan command. Maybe another package was needed?

Reply:
It's libyaml-tiny-perl